app/src/main/res/layout/drag_drop_test_fragment.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.recyclerview.widget.RecyclerView
+      android:id="@+id/drag_drop_recycler_view"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="1090dp"
```

Comment:
Why is this needed? Shouldn't this RecyclerView essentially be the same as the previous version?